src/main/java/com/conveyal/datatools/manager/jobs/ProcessSingleFeedJob.java

```diff
@@ -76,24 +78,21 @@ public String getFeedSourceId () {
      * zip file or the resulting database representation.
      *
-     * There are a few different transformation possibilities:
-     * 1. We only have zip transformations. These can either apply directly to the imported zip
-     *    or they can apply to a cloned version of the import. In the second case, the retrieval method should be set to
-     *    VERSION_CLONE (otherwise, it can be set to any other combination of retrieval methods).
-     * 2. We only have db transformations. These will always apply to a cloned snapshot of a feed version to
-     *    prevent the feed version's namespace from having different contents from the zip file. Following this, the
-     *    snapshot can either end up hanging (i.e., not published) or once all of the transformations have been
-     *    applied, we can create a new feed version from the snapshot (retrieval method being produced in house??).
-     * 3. Let's say we have zip transformations and db transformations.
-     *    a. Zip transformations apply to imported version, db transformations apply to new snapshot or get published as
-     *       additional feed version (no big deal).
-     *    b. Zip transformations apply to cloned version, db transformations apply to snapshot or get published.
-     *       This case is a little trickier, because we could end up with two cloned_versions. Let's say we want
-     *       a SQL query to run on the original data, create a new GTFS file, and then replace a GTFS+ file from some
-     *       other version.
+     * There are a few different transformation possibilities: 1. We only have zip transformations. These can either
+     * apply directly to the imported zip or they can apply to a cloned version of the import. In the second case, the
+     * retrieval method should be set to VERSION_CLONE (otherwise, it can be set to any other combination of retrieval
+     * methods). 2. We only have db transformations. These will always apply to a cloned snapshot of a feed version to
+     * prevent the feed version's namespace from having different contents from the zip file. Following this, the
+     * snapshot can either end up hanging (i.e., not published) or once all of the transformations have been applied, we
+     * can create a new feed version from the snapshot (retrieval method being produced in house??). 3. Let's say we
+     * have zip transformations and db transformations. a. Zip transformations apply to imported version, db
+     * transformations apply to new snapshot or get published as additional feed version (no big deal). b. Zip
+     * transformations apply to cloned version, db transformations apply to snapshot or get published. This case is a
+     * little trickier, because we could end up with two cloned_versions. Let's say we want a SQL query to run on the
+     * original data, create a new GTFS file, and then replace a GTFS+ file from some other version.
```

Comment:
Please revert the to the previous formatting.

Reply:
Updated.